test/prow/entrypoint.sh

```diff
@@ -22,5 +22,6 @@ echo "GEMINI_API_KEY=${GEMINI_API_KEY}" > .env
 
 cp $TEST_DIR/eval_data.yaml $TEMP_DIR/eval_data.yaml
 sed -i "s/uniq-cluster-name/${UNIQUE_ID}/g" $TEMP_DIR/eval_data.yaml
+sed -i "s|<workdir-placeholder>|${WORK_DIR}|g" $TEMP_DIR/eval_data.yaml
```

Comment:
Doing this only here (in the prow directory) makes these tests very hard to run locally. Can we either replace whatever would work locally or do this for both cases?